src/components/atoms/Button/style.ts

```diff
@@ -0,0 +1,23 @@
+import {StyleSheet} from 'react-native';
+import {colors} from '../../../static/consts/colors';
+
+export const styles = StyleSheet.create({
+  button: {
+    backgroundColor: colors.btnBackground,
+    height: 41,
+    width: 370,
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    marginBottom: 20,
+  },
+  buttonText: {
+    color: 'black',
+    fontSize: 20,
+    fontWeight: '400',
+  },
+  elevation: {
```

Comment:
Po co dodałeś to do dwóch osobnych styli? Przez to przekazujesz tablicę do button'a, która się redeklaruje na każdy rerender, a możesz to dodać do styli buttona od razu

Reply:
done